ui/app/adapters/pki/issuer.js

```diff
@@ -40,12 +81,25 @@ export default class PkiIssuerAdapter extends ApplicationAdapter {
   }
 
   query(store, type, query) {
-    return this.ajax(this.urlForQuery(query.backend), 'GET', this.optionsForQuery());
+    const { backend, shouldShowIssuerMetaData } = query;
+    const url = this.urlForQuery(backend);
+
+    return this.ajax(url, 'GET', this.optionsForQuery()).then(async (res) => {
+      // To show issuer meta data tags, we have a flag called shouldShowIssuerMetaData and only want to
```

Comment:
This flag is only sent when querying for the issuer's list view, right? I wonder if renaming it to something like `isListView` or `isIndexRoute` might help explain a little more about why/where this query request happens. We already know by the helpful function name below we're doing something about the metadata. 

Then it's pretty clear this conditional is saying "if it's the index route and there's only 10 then get the metadata!" 🔍  we might not even need the comments 🌟